tests/ui/cypress_specs/multi_user_config_file_with_ta.js

```diff
@@ -211,6 +211,9 @@ describe('Config File Users with TA', () => {
             if (ts.o) {
               cy.contains('Success')
               cy.contains('Error').should('not.exist')
+              // wait for the parent registration to complete inside Krill and
```

Comment:
This change is actually for #377 but was made here because #377 was blocking testing for #371.